# Python3 program to demonstrate
# the error in count() method

list1 = [1, 1, 1, 2, 3, 2, 1]

# Error when two parameters is passed.
print(list1.count(1, 2))
